### PR TITLE
Amending transcriptions openapi spec

### PR DESCRIPTION
**Pull request by davet1985**
### JIRA link (if applicable) ###

N/A

### Change description ###

- to include the extra property returning the POST /transcriptions 409 error

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/openapi/transcriptions.yaml [hunk 1]
```diff
@@ -60,8 +60,43 @@ paths:
                         $ref: '#/components/schemas/RequestTranscription400ErrorCode'
         '401':
           description: Unauthorised Error
+          content:
```
Comment (richardbull) [inline src/main/resources/openapi/transcriptions.yaml:63]: Don't think the 401 and 404 errors need expanding here.
They don't look like the standard problem error responses, the body is just empty.

401 is when the Azure access token has expired for example.
404 looks to be similar, so not the one you have added to the schema https://github.com/hmcts/darts-api/blob/master/src/main/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionController.java#L77

File: src/main/resources/openapi/transcriptions.yaml [hunk 2]
```diff
@@ -60,8 +60,43 @@ paths:
...
+            application/json+problem:
+              schema:
+                $ref: './problem.yaml'
+              example:
+                type: "AUTHORISATION_100"
+                title: "User is not authorised for the associated courthouse"
+                status: 401
```
Comment (richardbull) [inline src/main/resources/openapi/transcriptions.yaml:70]: I think this problem / example should be moved under 403 Forbidden Error - it looks to have been missed when @karen-hedges added the `@Authorisation` to the TranscriptionController

File: src/main/resources/openapi/transcriptions.yaml (hunk 1 above)
Comment (davet1985) [reply src/main/resources/openapi/transcriptions.yaml]: Removed.

File: src/main/resources/openapi/transcriptions.yaml (hunk 2 above)
Comment (davet1985) [reply src/main/resources/openapi/transcriptions.yaml]: Removed

File: src/main/resources/openapi/transcriptions.yaml [hunk 3]
```diff
@@ -469,7 +504,17 @@ components:
         - "TRANSCRIPTION_104"
         - "TRANSCRIPTION_106"
         - "TRANSCRIPTION_107"
-      x-enum-varnames: [ FAILED_TO_VALIDATE_TRANSCRIPTION_REQUEST, BAD_REQUEST_TRANSCRIPTION_TYPE, BAD_REQUEST_TRANSCRIPTION_URGENCY, DUPLICATE_TRANSCRIPTION ]
+      x-enum-varnames: [ FAILED_TO_VALIDATE_TRANSCRIPTION_REQUEST, BAD_REQUEST_TRANSCRIPTION_TYPE, BAD_REQUEST_TRANSCRIPTION_URGENCY ]
+    RequestTranscription404ErrorCode:
```
[the remark this answers is not in the document]
Comment (davet1985) [reply src/main/resources/openapi/transcriptions.yaml]: Removed

**Review by richardbull — APPROVED** [review]

File: src/main/resources/openapi/transcriptions.yaml (hunk 2 above)
Comment (davet1985) [reply src/main/resources/openapi/transcriptions.yaml]: Re-added back in the correct place.

**Pull request merged**